easy-ui-react/src/Menu/MenuItem.tsx

```diff
@@ -116,7 +115,7 @@ function LinkMenuItemContainer({
 }: ContainerProps) {
   return (
     <li role="none">
-      <Component ref={itemRef} {...props} onPointerUp={noop} onKeyDown={noop} />
```

Comment:
React Aria pushed a fix to their menu item hooks to no longer need requiring this for menu items that are links